apps/testingaccessibility/src/server/skill-next-auth-prisma-adapter.ts

```diff
@@ -0,0 +1,88 @@
+import type {PrismaClient, Prisma} from '../../generated/prisma/client'
+import type {Adapter} from 'next-auth/adapters'
+import {addSeconds, isAfter} from 'date-fns'
+
+export function PrismaAdapter(p: PrismaClient): Adapter {
```

Comment:
in practice this is very straight forward. there were a lot of TS errors, but it's a direct cut and paste and functions as expected